documentation/modules/deploying/proc-deploy-setup-external-clients.adoc

```diff
@@ -213,73 +219,62 @@ data:
-----
+kubectl get secret _<cluster_name>_-cluster-ca-cert -o jsonpath='{.data.ca\.crt}' | base64 -d > ca.crt
 
-. Extract the password from the password file:
+. Extract the user CA certificate from the `_<user_name>_` secret.
 +
 [source,shell,subs="+quotes"]
-----
-kubectl get secret _KAFKA-CLUSTER-NAME_-cluster-ca-cert -o jsonpath='{.data.ca\.password}' | base64 -d > ca.password
-----
-
-. Configure your client with the authentication details for the public cluster certificates:
-+
-.Sample client code
-[source,java,subs="+quotes,attributes+"]
-----
-properties.put("security.protocol","SSL"); <1>
-properties.put(SslConfigs.SSL_TRUSTSTORE_LOCATION_CONFIG,"/path/to/ca.p12"); <2>
-properties.put(SslConfigs.SSL_TRUSTSTORE_PASSWORD_CONFIG,_CA-PASSWORD_); <3>
-properties.put(SslConfigs.SSL_TRUSTSTORE_TYPE_CONFIG,"PKCS12"); <4>
-----
-<1> Enables TLS encryption (with or without TLS client authentication).
-<2> Specifies the truststore location where the certificates were imported.
-<3> Specifies the password for accessing the truststore. This property can be omitted if it is not needed by the truststore.
-<4> Identifies the truststore type.
-+
-NOTE: Use `security.protocol: SASL_SSL` when using SCRAM-SHA authentication over TLS.
+kubectl get secret _<user_name>_ -o jsonpath='{.data.user\.crt}' | base64 -d > user.crt
 
-. Extract the user CA certificate from the user Secret to the desired certificate format:
+. Extract the private key of the user from the `_<user_name>_` secret.
 +
 [source,shell,subs="+quotes"]
-----
-kubectl get secret _USER-NAME_ -o jsonpath='{.data.user\.p12}' | base64 -d > user.p12
-----
+kubectl get secret _<user_name>_ -o jsonpath='{.data.user\.key}' | base64 -d > user.key
 
-. Extract the password from the password file:
+. Configure your client with the bootstrap address hostname and port for connecting to the Kafka cluster:
 +
-[source,shell,subs="+quotes"]
+[source,env,subs="+quotes,attributes"]
 ----
-kubectl get secret _USER-NAME_ -o jsonpath='{.data.user\.password}' | base64 -d > user.password
+props.put(ConsumerConfig.BOOTSTRAP_SERVERS_CONFIG, "_<hostname>:<port>_");
 ----
 
-. Configure your client with the authentication details for the user CA certificate:
+. Configure your client with the truststore credentials to authenticate with the Kafka cluster.
++
+Specify the public cluster CA certificate.  
 +
-.Sample client code
-[source,java,subs="+quotes,attributes+"]
+.Example truststore configuration
+[source,env,subs="+quotes,attributes"]
 ----
-properties.put(SslConfigs.SSL_KEYSTORE_LOCATION_CONFIG,"/path/to/user.p12"); <1>
-properties.put(SslConfigs.SSL_KEYSTORE_PASSWORD_CONFIG,"<user.password>"); <2>
-properties.put(SslConfigs.SSL_KEYSTORE_TYPE_CONFIG,"PKCS12"); <3>
+props.put(CommonClientConfigs.SECURITY_PROTOCOL_CONFIG, "SSL");
+props.put(SslConfigs.SSL_TRUSTSTORE_TYPE_CONFIG, "PEM");
+props.put(SslConfigs.SSL_TRUSTSTORE_CERTIFICATES_CONFIG, "_<ca.crt_value>_");
```

Comment:
Is the `_value` here and bellow celar enough that it mean the content of the file? Should it be maybe something more like `_file_content`?